multiclusterhub/charts/search-prod/templates/api-service.yaml

```diff
@@ -0,0 +1,29 @@
+# Licensed Materials - Property of IBM
+# (C) Copyright IBM Corporation 2019 All Rights Reserved
+# US Government Users Restricted Rights - Use, duplication or disclosure restricted by GSA ADP Schedule Contract with IBM Corp.
+# Copyright (c) 2021 Red Hat, Inc.
+# Copyright Contributors to the Open Cluster Management project
+
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    service.beta.openshift.io/serving-cert-secret-name: search-api-certs
+  name: search-search-api
+  labels:
+    app: {{ .Values.search.name }}
+    component: "search-api"
+    chart: {{.Chart.Name}}-{{.Chart.Version}}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  ports:
+    - port: 4010
+      targetPort: 4010
+      protocol: TCP
+      name: http
+  selector:
+    app: {{ .Values.search.name }}
+    component: "search-api"
+    release: {{ .Release.Name }}
+  sessionAffinity: ClientIP
```

Comment:
Missing newline at end of last line.

https://stackoverflow.com/a/729795/1437822